src/textwatcher.js

```diff
@@ -79,8 +79,9 @@ export default class TextWatcher {
 	 *
 	 * @private
 	 * @param {'data'|'selection'} suffix Suffix used for generating event name.
+	 * @param {Object} data Data object for event.
 	 */
-	_evaluateTextBeforeSelection( suffix ) {
+	_evaluateTextBeforeSelection( suffix, data = {} ) {
```

Comment:
Do we still need this change?

Reply:
I'll check.

Reply:
We do pass the `batch` for the `matched:data` event.